src/lib/workspace/WorkspaceRouter.tsx

```diff
@@ -38,6 +47,11 @@ export function WorkspaceRouter({
           <Redirect to={location.pathname.replace(/\/\/+/g, '/')} />
         )}
       />
+      <Route
+        path={`${path}/studies`}
+        exact
+        render={() => <StudyList subsettingServiceUrl={subsettingServiceUrl} />}
```

Comment:
When called from this route, StudyList is rendering links with URLs like `/eda/studies/DS_841ea82`, but they just need to be `/eda/DS_841ea82`

Reply:
Ah, right. I will fix.